app/controllers/users/sessions_controller.rb

```diff
@@ -56,6 +56,8 @@ def timeout
       redirect_to root_url(request_id: request_id)
     end
 
+    def bounced; end
```

Comment:
It may make more send to put this in its own controller to keep the session controller clean and the views separated.